### ISSUE TITLE
[Select] label is not annouced

### ISSUE THREAD

**Issue by owestinTelus** [telusdigital/tds-core#436]
## Description
Dropdown <select> component does not announce the associated label. The select will call out the options, without context. 

## Reproduction Steps
- [ ]Using screenreader
- [ ] Tab (or click) the <select> component
- [ ] Results are announced
- [ ] label for <select> is not announced 

## Reproduction Percentage (if applicable)
- [ ] NA

## Version Number of TDS
- [ ] 0.3.0

*note attach screenshots whenever possible*
It took a while to find a resource that implements this correctly.
Here's a reference: [Deque's Pattern library <select>](https://pattern-library.dequelabs.com/components/selects

**on click**
<img width="405" alt="click with voice over on" src="https://user-images.githubusercontent.com/30598018/33568040-1d758136-d8f3-11e7-8764-489340fddd22.png">

**on keyboard focus**
<img width="416" alt="tab with voice over on" src="https://user-images.githubusercontent.com/30598018/33568041-1d848050-d8f3-11e7-9bfe-19e9d6a58d6b.png">

)

**Comment by ryanoglesby08** [comment]
I can confirm that this is indeed still valid.

However, I am having a very hard time finding any website that has a non-custom `<select>` in which the screenreader reads the label. Even here, the labels do not get read by Mac Voiceover: https://webaim.org/techniques/forms/controls#select.

@owestinTelus 
Thanks for sharing the link, https://pattern-library.dequelabs.com/components/selects. But, that is an example of a completely custom dropdown, which does not apply to our case, as we currently use the browser's standard dropdown, implemented as a `<select>`.

I have even tried other `aria-*` properties such as `aria-labelledby` with no success. Am I missing something here Oskar?

**Comment by owestinTelus** [comment]
@ryanoglesby08 I am not able to help you with the solution. As noted, I have only seen the Dequelabs example.

I think that we can close this issue or flag it as low priority.

We can revisit this when we hire an accessibility specialist, to provide additional technical support.

**Comment by ryanoglesby08** [comment]
Ok, closing as per Oskar's recommendation. This behaviour seems to be the norm and could be addressed if/when we create a completely custom dropdown.
